conf/ocsci/default_config.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Btw why this can't be in `conf/` directory directly?

Reply:
I wanted to separate ocsci configuration here, there are going to be a lot of configuration for different cluster configurations we are going to have. So we will need to somehow structure them. It will be mess when we will mix those configurations in one folder I think.

Reply:
Ah, ok.